hokusai/staging.yml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: opstools-terraform-drift-detection
+spec:
+  schedule: "15 7 * * *"
```

Comment:
Daily at 7:15am UTC. Seems important to be outside of business hours (Berlin, NY), to reduce alert noise about temporary divergence between `main` and branches (e.g. Infrastructure repo) on Terraform plans.